src/lm.jl

```diff
@@ -200,7 +203,7 @@ deviance(obj::LinearModel) = deviance(obj.rr)
 
 For linear models, the deviance of the null model is equal to the total sum of squares (TSS).
 """
-nulldeviance(obj::LinearModel) = nulldeviance(obj.rr)
+nulldeviance(obj::LinearModel) = nulldeviance(obj.rr, hasintercept(obj))
 loglikelihood(obj::LinearModel) = loglikelihood(obj.rr)
 nullloglikelihood(obj::LinearModel) = nullloglikelihood(obj.rr)
```

Comment:
Also change the definition of `nullloglikelihood` for consistency.

Reply:
Okay.